test_src/sljitTest.c

```diff
@@ -11555,7 +11556,6 @@ static void test92(void)
 
 	sljit_emit_enter(compiler, 0, SLJIT_ARGS1(VOID, P), 5, 5, 0, 0, 2 * sizeof(sljit_sw));
 
-	sljit_emit_op1(compiler, SLJIT_MOV, SLJIT_MEM1(SLJIT_S0), sizeof(sljit_sw), SLJIT_IMM, -9856);
```

Comment:
Why is this removed? It checks that you can write data before sljit_emit_atomic_load

Reply:
sljit_emit_atomic_load() can't prevent any write operations before it is even executed

Reply:
also, it looked weird when reviewing the generated code, as the same memory address was being overwritten a few instructions later

Reply:
That is the whole point! The memory can be initialized one instruction earlier, and it must still work.